### ISSUE TITLE
Airdates wrong / theTVdb no connecting for new shows??

### ISSUE THREAD

**Issue by mattpulpfree** [SickChill/SickChill#4006]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: Master / 832ccb445f8eaadef783eddb51880d67d1a77b3d
OS: Windows Server 2008 R2
What happened: air-dates are wrong, even after trying to update shows

I'm hoping someone can help here, for the last month or so i've notied that air-dates seem to be off on some of my shows ( south park, broad city, rick and morty) . It seems like i'm not getting info from thetvdb properly maybe?

**Comment by mattpulpfree** [comment]
in fact i'm seeing now that the last log was from 7 / 28 / 2017...unless the date is off on my sickrage somehow, server shows 8/28 for the date?

**Comment by mmazur519** [comment]
Having the same issue.
Log:
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Mindhunter
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Submission
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Leah Remini: Scientology and the Aftermath is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for The Good Doctor
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for The Path
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Mary Kills People
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for The Magicians (2015)
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Absentia
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Midnight, Texas is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Friends from College
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Bosch
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Salvation
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for In The Dark
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Doubt
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Rick and Morty is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Rosewood is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Black Mirror is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Great News is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Playing House
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Wrecked (2016) is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Dirty Jobs
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: The Mist is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Young Sheldon
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: American Gods is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Imposters
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Orange Is the New Black
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Kicking & Screaming
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Trial & Error
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Big Little Lies
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for The Good Fight
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for MasterChef Canada
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Girls
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Workin' Moms
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Prison Break
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Shut Eye
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: This Is Us is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for American Housewife
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Kevin Can Wait
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Penny Dreadful
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for The Amazing Race Canada
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Lucifer
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Houdini & Doyle
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Preacher is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for The Girlfriend Experience
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Schitt's Creek
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Shark Tank
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Highway Thru Hell
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Young & Hungry
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Suits is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: The Simpsons is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Reign (2013)
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for The Profit
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Once Upon a Time (2011) is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Queueing show refresh for Modern Family
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: iZombie is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Gotham is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: A refresh was attempted but there is already an update queued or in progress. Updates do a refresh at the end so I'm skipping this request.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: The Big Bang Theory is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: Ballers is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: American Dad! is in process of being updated by Post-processor or manually started, can't update again until it's done.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: Automatic update failed: This show is already being refreshed, not refreshing again.
2017-08-29 12:18:14 DEBUG SHOWUPDATER :: GET URL: http://thetvdb.com/api/Updates.php?type=series&time=1503937080 [Status: 200]

[img]http://i.imgur.com/ymiZrNQ.jpg[/img]

The system schedules the updates to the shows but it isn't actually completeing any of the updates that it has queued up.

**Comment by mmazur519** [comment]
Will update fine if done manually. But if there is a failure in the queueing system it seems to crash and can never recover.

**Comment by mattpulpfree** [comment]
I ended up fixing my issue by restarting my whole server. I used to get a similar issue where things wouldn't queue properly, couldn't run backlog or manually search episodes and would tell me it couldn't find any episodes even though when I did a search on my own, tons of possibilities would show up. not sure why but restarting always fixes it for me. Just had been awhile since that happened and have never had shows have bad air-dates and not update when forcing. 

quite odd.
